pkg/crc/machine/bundle/constants.go

```diff
@@ -8,23 +8,23 @@ import (
 var bundleLocations = map[string]bundlesDownloadInfo{
 	"amd64": {
 		"darwin": {
-			preset.OpenShift: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/4.9.18/crc_hyperkit_4.9.18.crcbundle",
-				"d354e7c9bfb88abfb822151bf721d1df45a83b4ac0330f3f5c2bb930466ae86d"),
-			preset.Podman: download.NewRemoteFile("https://storage.googleapis.com/crc-bundle-github-ci/3.4.4-1/crc_podman_hyperkit_3.4.4_amd64.crcbundle",
-				"1195daa0bccd6293609673f424c788b7ad153a44829d55168460370fb88920e1"),
+			preset.OpenShift: download.NewRemoteFile("https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/openshift/4.10.3/crc_hyperkit_4.10.3_amd64.crcbundle",
+				"57c8adae49beeb83d7a180aaba962256c2833863e1f59492295841a3e6e3f016"),
+			preset.Podman: download.NewRemoteFile("https://mirror.openshift.com/pub/openshift-v4/clients/crc/bundles/podman/3.4.4/crc_podman_hyperkit_3.4.4_amd64.crcbundle",
+				"12c0b610e8e3a0d446ac106e2b6f4345ee6fc45304ed05d529933a1261a6dd04"),
```

Comment:
Fwiw, we should avoid distributing files with the same name (`crc_podman_hyperkit_3.4.4_amd64.crcbundle`), but different checksums